fingerprintauthhelper/src/main/kotlin/com/pro100svitlo/fingerprintAuthHelper/FahManager.kt

```diff
@@ -123,96 +108,94 @@ internal class FahManager(@NonNull c: Context, l: FahListener?, keyName: String,
         logThis("error: " + FahErrorType.getErrorNameByCode(FahErrorType.HELP_ERROR_BASE + helpMsgId) +
                 " (" + helpString + ")")
 
-        mListener?.get()?.onFingerprintStatus( false, FahErrorType.HELP_ERROR_BASE + helpMsgId, helpString)
+        listener?.get()?.onFingerprintStatus( false, FahErrorType.HELP_ERROR_BASE + helpMsgId, helpString)
     }
 
     override fun onAuthenticationFailed() {
         logThis("AUTH_NOT_RECOGNIZED")
         mTriesCountLeft--
-        mListener?.get()?.onFingerprintStatus(false, FahErrorType.Auth.AUTH_NOT_RECOGNIZED,
-                mContext?.get()?.getString(R.string.FINGERPRINT_NOT_RECOGNIZED))
+        listener?.get()?.onFingerprintStatus(false, FahErrorType.Auth.AUTH_NOT_RECOGNIZED,
+                context.get()?.getString(R.string.FINGERPRINT_NOT_RECOGNIZED))
     }
 
     override fun onAuthenticationSucceeded(result: FingerprintManager.AuthenticationResult) {
         logThis("onAuthenticationSucceeded")
         mTriesCountLeft = TRY_LEFT_DEFAULT
-        mListener?.get()?.onFingerprintStatus(true, -1, "")
+        listener?.get()?.onFingerprintStatus(true, -1, "")
     }
 
+    @TargetApi(Build.VERSION_CODES.M)
     internal fun startListening(): Boolean {
-        mIsActivityForeground = true
+        isActivityForeground = true
         if (mTimeOutLeft > 0 || !canListen(true) || !initCipher()) {
-            mIsListening = false
+            isListening = false
             mTriesCountLeft = 0
         } else {
-            mAfterStartListenTimeOut = true
-            Handler().postDelayed({ mAfterStartListenTimeOut = false }, 200)
-            mCancellationSignal = CancellationSignal()
-            mSelfCancelled = false
+            afterStartListenTimeOut = true
+            Handler().postDelayed({ afterStartListenTimeOut = false }, 200)
+            cancellationSignal = CancellationSignal()
+            selfCancelled = false
             // The line below prevents the false positive inspection from Android Studio
 
-            mFingerprintManager?.authenticate(mCryptoObject, mCancellationSignal, 0 /* flags */, this, null)
-            mListener?.get()?.onFingerprintListening(true, 0)
-            mIsListening = true
+            fingerprintManager.authenticate(cryptoObject, cancellationSignal, 0 /* flags */, this, null)
+            listener?.get()?.onFingerprintListening(true, 0)
+            isListening = true
             mTriesCountLeft = TRY_LEFT_DEFAULT
         }
         registerBroadcast(true)
-        return mIsListening
+        return isListening
     }
 
     internal fun stopListening(): Boolean {
-        mIsActivityForeground = false
-        if (mCancellationSignal != null) {
-            mSelfCancelled = true
-            mCancellationSignal?.cancel()
-            mCancellationSignal = null
-            mIsListening = false
+        isActivityForeground = false
+        if (cancellationSignal != null) {
```

Comment:
`if (cancellationSignal != null) {`
this one also can be changed for `cancellationSignal?.le{`